assets/src/design-system/theme/global/overrideFocusOutline.js

```diff
@@ -20,6 +20,10 @@
 import { createGlobalStyle, css } from 'styled-components';
 import 'focus-visible';
 
+import(/* webpackChunkName: "focus-visible" */ 'focus-visible').catch(
```

Comment:
Saw you mentioned lazy loading this in the description, should it also have `webpackMode: "lazy"` then?

Reply:
That's the [default](https://webpack.js.org/api/module-methods/#magic-comments)